<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-Aditya" failures="1" tests="1" name="Test1" time="64.679" errors="0" timestamp="2025-06-20T17:48:39 IST">
  <testcase classname="contactTests.ToCreateContactTests" name="toCreateContact_001" time="38.887">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at contactTests.ToCreateContactTests.toCreateContact_001(ToCreateContactTests.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- toCreateContact_001 -->
</testsuite> <!-- Test1 -->
